src/Event/Subscriber/ProgressSubscriber.php

```diff
@@ -34,6 +35,10 @@ class ProgressSubscriber implements EventSubscriberInterface
      */
     public function __construct(OutputInterface $output, ProgressBar $progressBar)
     {
+        if ($output instanceof ConsoleOutputInterface) {
```

Comment:
Shouldn't this be fixed here instead?
https://github.com/phpro/grumphp/blob/master/src/Console/Helper/TaskRunnerHelper.php#L96

Reply:
I also agree with @veewee